README.md

```diff
@@ -796,14 +796,7 @@ The `size_t malloc_object_size(void *ptr)` function returns an *upper bound* on
 the accessible size of the relevant object (if any) by querying the malloc
 implementation. It's similar to the `__builtin_object_size` intrinsic used by
 `_FORTIFY_SOURCE` but via dynamically querying the malloc implementation rather
-than determining constant sizes at compile-time. The current implementation is
-just a naive placeholder returning much looser upper bounds than the intended
-implementation. It's a valid implementation of the API already, but it will
-become fully accurate once it's finished. This function is **not** currently
```

Comment:
The documentation on signal safety should be left here since this hasn't been addressed like before yet.